cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -313,7 +313,7 @@ func (s *CMServer) Run(_ []string) error {
 	// If apiserver is not running we should wait for some time and fail only then. This is particularly
 	// important when we start apiserver and controller manager at the same time.
 	var versionStrings []string
-	err = wait.PollImmediate(time.Second, 10*time.Second, func() (bool, error) {
+	err = wait.Poll(time.Second, 10*time.Second, func() (bool, error) {
 		versionStrings, err = client.ServerAPIVersions(kubeconfig)
 		return err == nil, err
```

Comment:
Thanks for fixing this.  I was told to use PollImmediate without actually looking at it.  The difference between Poll and PollImmediate is that PollImmediate will do a pre-wait check, so they essentially have the same effect here.  The check "err == nil" alreay tells Poll to continue checking, so I think it's safe to return err == nil, err.